rules/windows/registry/registry_event/registry_event_malware_qakbot_registry.yml

```diff
@@ -0,0 +1,21 @@
+title: Potential Qakbot Registry Activity
+id: 1c8e96cd-2bed-487d-9de0-b46c90cade56
+status: experimental
+description: Detects a registry key used by IceID in a campaign that distributes malicious OneNote files
+references:
+    - https://www.zscaler.com/blogs/security-research/onenote-growing-threat-malware-distribution
+author: Hieu Tran
+date: 2023/03/13
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection:
+        TargetObject|endswith: '\Software\firm\soft'
```

Comment:
```suggestion
        TargetObject|endswith: '\Software\firm\soft\'
```
at least add the `\`. If I didn't misread the report is has to be there as there is a subkey `\name`
![image](https://user-images.githubusercontent.com/79651203/225875575-b2937422-00ce-47b6-8b55-8d51f93a1c45.png)

Reply:
![image](https://user-images.githubusercontent.com/8741929/225885390-8042180b-40b7-4e0f-8252-fd4b2960d5c9.png)